scripts/loud.coffee

```diff
@@ -21,6 +21,9 @@ module.exports = (robot) ->
   robot.hear /^([A-Z"][A-Z0-9 .,'"()\?!&%$#@+-]+)$/, (res) ->
     # Pick a loud from the stored list and say it. Skip if there are no louds.
     old_loud = res.random(robot.brain.get('louds'))
+    if old_loud in robot.brain.get('louds_banned')
```

Comment:
I say just skip retrying. If we get a bad loud, we can just ban it interactively. 

Not worth the extra branch IMO